src/main/resources/lib/credentials/select.jelly

```diff
@@ -107,11 +107,11 @@
           <input type="hidden" name="includeUser" value="${includeUser}"/>
         </j:otherwise>
       </j:choose>
-      <div class="form-group jenkins-form-item jenkins-form-item--tight">
-        <f:select clazz="${attrs.clazz} credentials-select" field="${attrs.field}"
-                  default="${paramDefault?'':attrs.default}" value="${paramValue?(paramDefault?'':attrs.default):value}"
-                  checkMethod="${attrs.checkMethod}"/>
-      </div>
+      <!-- Placing this in a div will block the field validation. The attrs.checkMethod is not called -->
```

Comment:
this is not correct, it's one of the classes, anyway I'll push something to fix it up